### PR TITLE
Created base functionality to render service info in the InfoDropdown…

### PR DESCRIPTION
**Pull request by dannyscheiner**
… component, as well as drilled data from state into the Service component

### CODE REVIEW

File: src/renderer/App.d.ts [hunk 1]
```diff
@@ -47,6 +47,26 @@ type Graph = {
...
+  working_dir?: string;
+  security_opt?: string[];
+  stop_grace_Period?: string;
+  stop_signal?: string;
+  tmpfs?: string[];
+  restart: string;
+  pid: string;
+};
+
+export type ServiceInfo = {
```
Comment (jdnordy) [inline src/renderer/App.d.ts]: @dannyscheiner I think this type name should match what it's typing in state.
```
export type Services = {
  [service: string]: Service;
};
```
Then on line 6: 
`services: Services;`

File: src/renderer/App.tsx [hunk 2]
```diff
@@ -127,6 +129,7 @@ class App extends Component<{}, State> {
         <D3Wrapper
           fileUploaded={this.state.fileUploaded}
           fileUpload={this.fileUpload}
+          serviceInfo={this.state.services}
```
Comment (jdnordy) [inline src/renderer/App.tsx]: Same as previous comment: The props should have the same designation as the state.
`          services={this.state.services}`

File: src/renderer/components/D3Wrapper.tsx [hunk 3]
```diff
@@ -15,17 +15,22 @@ import HostOS from './HostOS';
 import FileSelector from './FileUpload';
 
 //type import
-import { FileUpload } from '../App.d';
+import { FileUpload, ServiceInfo } from '../App.d';
```
Comment (jdnordy) [inline src/renderer/components/D3Wrapper.tsx]: Same as previous comments.

File: src/renderer/components/D3Wrapper.tsx [hunk 4]
```diff
@@ -15,17 +15,22 @@ import HostOS from './HostOS';
...
   fileUpload: FileUpload;
   fileUploaded: boolean;
+  serviceInfo: ServiceInfo;
 };
 
-const D3Wrapper: React.FC<Props> = ({ fileUploaded, fileUpload }) => {
+const D3Wrapper: React.FC<Props> = ({
+  fileUploaded,
+  fileUpload,
+  serviceInfo,
```
Comment (jdnordy) [inline src/renderer/components/D3Wrapper.tsx]: flag

File: src/renderer/components/D3Wrapper.tsx [hunk 5]
```diff
@@ -15,17 +15,22 @@ import HostOS from './HostOS';
...
+}) => {
   return (
     <div className="d3-wrapper">
-      <ServicesWrapper />
+      <ServicesWrapper serviceInfo={serviceInfo} />
```
Comment (jdnordy) [inline src/renderer/components/D3Wrapper.tsx]: services

File: src/renderer/components/InfoDropdown.tsx [hunk 6]
```diff
@@ -13,9 +13,84 @@ import { FaAngleDown } from 'react-icons/fa';
 import Accordion from 'react-bootstrap/Accordion';
 import Card from 'react-bootstrap/Card';
 
-type Props = {};
+// import { Service } from '../App.d';
+
+type Props = {
+  service?: any;
```
Comment (jdnordy) [inline src/renderer/components/InfoDropdown.tsx:19]: Shouldn't this have type: Service

File: src/renderer/components/ServicesWrapper.tsx [hunk 7]
```diff
@@ -9,15 +9,20 @@
...
-//import Services from './Service';
+import Service from './Service';
 import DependsOnView from './DependsOnView';
-type Props = {};
 
-const ServicesWrapper: React.FC<Props> = props => {
+import { ServiceInfo } from '../App.d';
+
+type Props = {
+  serviceInfo: ServiceInfo;
```
Comment (jdnordy) [inline src/renderer/components/ServicesWrapper.tsx]: serviceInfo should be services

**Review by jdnordy — CHANGES_REQUESTED** [review]
Needs changes before merge.

File: src/renderer/components/InfoDropdown.tsx (hunk 6 above)
Comment (michaelbayday) [reply src/renderer/components/InfoDropdown.tsx:19]: doesn't work with Service

Comment (michaelbayday) [reply src/renderer/components/InfoDropdown.tsx:19]: i think you can help with resolving the type maybe haha

**Review by jdnordy — APPROVED** [review]

File: src/renderer/App.tsx [hunk 8]
```diff
@@ -27,7 +27,7 @@ import D3Wrapper from './components/D3Wrapper';
 import { State, FileUpload, UpdateOption } from './App.d';
 
 const initialState: State = {
-  selectedContainer: '',
+  selectedContainer: 'app',
```
Comment (michaelbayday) [inline src/renderer/App.tsx:30]: hardcoded for now

**Pull request merged**